### PR TITLE
Opinionated design changes on partner sup form

### PR DESCRIPTION
**Pull request by exu3**
This PR does the following (somewhat related to #2206)
- Remove dark theme toggle
  - The intention is to have this just in light-mode, since it feels better integrated with Givebutter's design that way. However, [this](https://github.com/hackclub/bank/blob/e1f789bf88508bcc728deefef988335b1b75c472/app/assets/javascripts/global.js#L41) sets the page color mode to the browser's `prefers-color-scheme` so it's a bit broken there.
  - What's the best way to go about forcing a color theme?
- Adds an animated background gradient similar to apply.hackclub.com
- Changes “open an account" to “become a nonprofit”

Thoughts on these changes?

--- 

<details>
<summary>Here's what it looks like in light mode currently</summary>
!

</details>

### CODE REVIEW

**Comment by maxwofford** [comment]
Love the blending of GB & HCB's theming-- this is def an improvement

**Comment by maxwofford** [comment]
Couple comments:
- Let's have a quick section that reiterates the features they get (super simple list ex. cards, direct deposit, invoices)
- Let's reiterate the fee
- The footer link should link to the givebutter version of the marketing page, or the givebutter help center page

For inspiration, [this is an absolutely kickass form](https://givebutter.com/fiscal-sponsorship) & I'd love to have the same vibe:
Screen Shot 2022-02-01 at 09 51 51

**Comment by exu3** [comment]
https://bank--pr-2281.herokuapp.com/partnered_signups/sup_5xvtJX

**Comment by maxwofford** [comment]
@exu3 I brought up a couple comments IRL to @MelSmith104 about the form. Let me know if you're still looking for review on this & i'll be sure to make time for it during my bank day tomorrow.

**Comment by exu3** [comment]
Ah gotcha, I don't think I got the comments about this form but I'll check with Mel again & let you know if I still need a review tomorrow. Thanks!

**Comment by exu3** [comment]
Hi @maxwofford, if you have a chance today, it would be awesome if you could give this a review. I wasn't able to get any comments from Mel.


https://bank--pr-2281.herokuapp.com/partnered_signups/sup_5xvtJX

File: app/views/partnered_signups/_form.html.erb [hunk 1]
```diff
@@ -25,7 +25,9 @@
     <%= form.label :owner_birthdate, "Date of birth" %>
     <%= form.date_field :owner_birthdate, class: 'fit', style: 'width: min-content', value: @partnered_signup.owner_birthdate %>
   </p>
-
+  <p class="flex" style="flex-direction: row-reverse; margin-right: auto;">
+    <%= form.label :legal_acknowledgement, "I ackgnowlakjdf;lakjfd;klasjdf;laksdjf;askjf;askdj", required: true %>
```
Comment (maxwofford) [inline app/views/partnered_signups/_form.html.erb]: lol

**Comment by maxwofford** [comment]
Once the stuff in the video is done lmk & we can merge-- this is def ready.

**Comment by exu3** [comment]
@maxwofford I think I hit everything except the country field. Right now, it's an integer; would a reasonable fix for this be to change it to a string because most (if not all) signups will have US addresses anyways?

File: app/views/partnered_signups/_form.html.erb (hunk 1 above)
Comment (exu3) [reply app/views/partnered_signups/_form.html.erb]: POV: you're about to be late to class.

**Comment by maxwofford** [comment]
@exu3 could you make it a dropdown like events have?

**Comment by maxwofford** [comment]
This looks great! Good to merge. I opened https://github.com/hackclub/bank/pull/2300 to help clean up the cross-dependencies, but I'm cool getting that merged in later

**Pull request merged**